docs/user-guide/config-examples.md

```diff
@@ -0,0 +1,30 @@
+# User configurations
+
+Want some help with starting your configuration ?
+Have a look at this catalog of configuration examples.
+
+<!-- TODO: add info about `kickstart.nixvim` -->
+
+**Note:**\
+Most of those configurations are using a [standalone flake](../modules/standalone.html).\
+Of course, all of the nixvim options are accessible no matter how you are using it (flake, NixOS/HM module, nix-darwin...).
+
+| Config                                                                                                    | Comment                                               |
+|-----------------------------------------------------------------------------------------------------------|-------------------------------------------------------|
+| [ahwxorg/nixvim-config](https://github.com/ahwxorg/nixvim-config)                                         |                                                       |
+| [alisonjenkins/neovim-nix-flake](https://github.com/alisonjenkins/neovim-nix-flake)                       |                                                       |
+| [GaetanLepage/nix-config](https://github.com/GaetanLepage/nix-config/tree/master/home/modules/tui/neovim) | Home-manager                                          |
+| [gwg313/nvim-nix](https://github.com/gwg313/nvim-nix)                                                     |                                                       |
+| [hbjydev/hvim](https://github.com/hbjydev/hvim)                                                           |                                                       |
+| [nicolas-goudry/nixvim-config](https://github.com/nicolas-goudry/nixvim-config)                           | Heavily inspired by AstroNvim                         |
+| [pete3n/nixvim-flake](https://github.com/pete3n/nixvim-flake)                                             |                                                       |
+| [redyf/Neve](https://github.com/redyf/Neve)                                                               | meticulously crafted custom configuration for Nixvim  |
+| [siph/nixvim-flake](https://github.com/siph/nixvim-flake)                                                 |                                                       |
+| [Tanish2002/neovim-config](https://github.com/Tanish2002/neovim-config)                                   |                                                       |
+| [traxys/Nixfiles](https://github.com/traxys/Nixfiles/tree/master/neovim)                                  |                                                       |
```

Comment:
Have you considered whether tables are suitable when reading the docs on mobile (or a small desktop window)?

Reply:
If we go with a table (they do look nice), it may be wise to _either_ increase the column width a bit, _or_ don't align the column dividers at all (`|`). That way adding a new row with a wider column in the future shouldn't affect _all the other rows_ as well.

I'd also suggest using footnote-style markdown link syntax here, to make the source code a little more readable and less wide.

E.g. with un-aligned columns and footnote links:

```markdown
| Config | Comment |
|-|-|
| [siph/nixvim-flake] | |
| [Tanish2002/neovim-config] | |
| [traxys/Nixfiles] | |

[siph/nixvim-flake]: https://github.com/siph/nixvim-flake
[Tanish2002/neovim-config]: https://github.com/Tanish2002/neovim-config
[traxys/Nixfiles]: https://github.com/traxys/Nixfiles/tree/master/neovim
```

Reply:
I didn't know about this syntax. Done !